atomics/T1036/T1036.yaml

```diff
@@ -11,4 +11,30 @@ atomic_tests:
       copy %WINDIR%\System32\cmd.exe /Y %ALLUSERSPROFILE%\cmd.exe
       start %ALLUSERSPROFILE%\cmd.exe
     cleanup_command: del %ALLUSERSPROFILE%\cmd.exe >nul 2>&1
-    name: command_prompt
\ No newline at end of file
+    name: command_prompt
+- name: Malware Masquerading and Execution from Zip File
+  description: Upon the unzipping of the file, the .cmd file executes and changes the .pdf to .dll and executes the dll

```

Comment:
Could you add the link to the blog post that describes this TTP and the APT group who uses it to the description of this atomic?